### ISSUE TITLE
run on RTX5090

### ISSUE THREAD

**Issue by junyiwuu** [NVlabs/nvdiffrec#170]
Hi, this is not an issue, but the issue that I met and spent long time to solve, just in case if anyone met the same problem in the future:

Since this repo is using old version of python, torch, tiny-cuda-nn, but I want to use the latest everything.  The major issue comes from tiny-cuda-nn

GPU: RTX5090
cuda 12.8
Rocky Linux

How to run Nvdiffrec:
(create a new conda env, in your conda environment)
1. install python3.12
2.  latest stable version torch: `pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu128`
3. all other requirements in the readme, **except the line for tiny-cuda-nn**
4. `export TCNN_CUDA_ARCHITECTURES=120`
5. `pip install git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch`
6. Find all `torch.cross` in the repo code, add `, dim=-1` in the end
7. I have opengl context issue, so I changed (in train.py)  `glctx = dr.RasterizeGLContext()`  to  `glctx = dr.RasterizeCudaContext()`

Then everything finally work
